config/ldes-delta-pusher/handle-regular-types.ts

```diff
@@ -0,0 +1,80 @@
+import { Changeset } from "../types";
+import { query } from "mu";
+import { publish } from "./publisher";
+import { log } from "./logger";
+
+type InterestingSubject = {
+  uri: string;
+  ldesType: string;
+};
+
+const regularTypesToLDESMapping = {
+  "http://data.vlaanderen.be/ns/mandaat#Fractie": "fractie",
+  "http://www.w3.org/ns/org#Membership": "lidmaatschap",
+  "http://data.vlaanderen.be/ns/mandaat#Mandaat": "mandaat",
+  "http://www.w3.org/ns/person#Person": "persoon",
+  "http://purl.org/dc/terms/PeriodOfTime": "periode",
+  "http://purl.org/dc/terms/Identifier": "identifier",
```

Comment:
is this about the rijksregisternummer? if so, this should be identificator.

Reply:
hm it's not actually the right part that's wrong (it's just the name of the ldes stream we're creating) but the left part, which uses the wrong prefix!